blenderproc/python/writer/BopWriterUtility.py

```diff
@@ -131,14 +132,41 @@ def write_bop(output_dir: str, target_objects: Optional[List[MeshObject]] = None
         chunk_dir_ids = [d.split('/')[-1] for d in chunk_dirs]
         chunk_dirs = chunk_dirs[chunk_dir_ids.index(f"{starting_chunk_id:06d}"):]
 
+        # convert all objects to trimesh objects
+        trimesh_objects = {}
+        for obj in dataset_objects:
+            trimesh_obj = obj.mesh_as_trimesh()
+            if m2mm:
+                trimesh_obj.apply_scale(scaling=0.001)
+            trimesh_objects[obj.get_cp('category_id')] = pyrender.Mesh.from_trimesh(mesh=trimesh_obj)
+
         _BopWriterUtility.calc_gt_masks(chunk_dirs=chunk_dirs, starting_frame_id=starting_frame_id,
-                                        dataset_objects=dataset_objects, delta=delta)
+                                        dataset_objects=trimesh_objects, m2mm=m2mm, delta=delta)
         _BopWriterUtility.calc_gt_info(chunk_dirs=chunk_dirs, starting_frame_id=starting_frame_id,
-                                       dataset_objects=dataset_objects, delta=delta)
+                                       dataset_objects=trimesh_objects, m2mm=m2mm, delta=delta)
         _BopWriterUtility.calc_gt_coco(chunk_dirs=chunk_dirs, dataset_objects=dataset_objects,
                                        starting_frame_id=starting_frame_id)
 
 
+def bop_pose_to_pyrender_coordinate_system(cam_R_m2c: np.ndarray, cam_t_m2c: np.ndarray) -> np.ndarray:
+    """ Converts an object pose in bop format to pyrender coordinate system
+        (https://pyrender.readthedocs.io/en/latest/examples/cameras.html).
+
+    :param cam_R_m2c: 3x3 Rotation matrix.
+    :param cam_t_m2c: Translation vector.
+    :return: Pose in pyrender coordinate system.
+    """
+    # create homogeneous transformation matrix
+    bop_pose = np.eye(4)
+    bop_pose[:3, :3] = cam_R_m2c
+    bop_pose[:3, 3] = cam_t_m2c
+
+    # flip z-axis
+    rot = np.eye(4)
+    rot[1, 1], rot[2, 2] = -1, -1
+    return rot.dot(bop_pose)
```

Comment:
Can we use here the already existing utility function, to make it more readable:
```suggestion
    return change_target_coordinate_frame_of_transformation_matrix(bop_pose, ["X", "-Y", "-Z"])
```

Needs the import:
```
from blenderproc.python.utility.MathUtility import change_target_coordinate_frame_of_transformation_matrix
```